buildtools/fsyacc/fsyaccdriver.fs

```diff
@@ -199,6 +199,8 @@ let writeSpecToFile (generatorState: GeneratorState) (spec: ParserSpec) (compile
               writer.WriteLineInterface "module %s" s;
       
       writer.WriteLine "#nowarn \"64\";; // turn off warnings that type variables used in production annotations are instantiated to concrete type";
+      writer.WriteLine "#nowarn \"1182\"  // the generated code often has unused variable 'parseState'"
```

Comment:
This change should also be made in https://github.com/fsprojects/FsLexYacc/

Reply:
Created https://github.com/fsprojects/FsLexYacc/pull/218